syft/messaging/message.py

```diff
@@ -121,6 +123,14 @@ def communication(name, target, args_, kwargs_, return_ids):
         action = CommunicationAction(name, target, args_, kwargs_, return_ids)
         return TensorCommandMessage(action)
 
+    @staticmethod
+    def workerAction(name, target, args_, kwargs_, return_ids):
```

Comment:
You must be writing other languages right now too! 😄 
```suggestion
    def worker_action(name, target, args_, kwargs_, return_ids):
```